### ISSUE TITLE
Update Readme in Monorepo with all package versions

### ISSUE THREAD

**Issue by ycrc5** [googleapis/release-please-action#779]
### TL;DR

I see there is an option for adding extra-files, but the annotations are generic which means I would not be able to update different package information in a single file.

### Detailed design

From release please documentation, I see that you can add a file and annotate the line to change as follow:
```
-- x-release-please-start-version
obj.version = "1.0"
-- x-release-please-end
```

However, there does not seem to be a context awareness in the case where I have 1 readme file that contains multiple entries for different packages.

The idea here is to have the root readme contain a table with information of all the services. Something like:

| Service Name | Description | Version |
| --- | --- | --- | 
| service-a | - | 1.4.5 |
| service-b | - | 2.2.1 |

And then be able to annotate the line to target as follow:
```
| service-a | - | 1.4.5 | -- x-release-please-start-version-service-a
```

Is there currently any way to achieve this?

**Comment by chingor13** [comment]
Unfortunately this is not supported. This would need to be implemented on the upstream https://github.com/googleapis/release-please library
